css-layout-classes/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+
+      <!-- link image -->
+      <div class="row">
+        <div class="column-third">
+          <img class="character" src="./images/link.png" alt="link picture" />
+        </div>
+        <!-- super smash text content -->
+        <div class="column-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series. It received praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <!-- yoshi image -->
+        <div class="column-third">
+          <img class="character" src="./images/yoshi.png" alt="yoshi picture" />
+        </div>
+      </div>
+
+      <!-- kirby image -->
+      <div class="row">
+        <div class="column-fourth">
+          <img class="character" src="./images/kirby.png" alt="kirby picture" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-fourth">
+          <img class="character" src="./images/fox.png" alt="fox picture" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-fourth">
+          <img
+            class="character"
+            src="./images/pikachu.png"
+            alt="pikachu picture" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-fourth">
+          <img class="character" src="./images/ness.png" alt="ness picture" />
+        </div>
+      </div>
```

Comment:
All of this should be inside the same `row`. You shouldn't have multiple `row`s here. It's just 1 `row` & 4 `column-fourth` as children